examples/index.md

```diff
@@ -179,63 +181,26 @@ Content-Type: application/vnd.api+json
   "errors": [
     {
       "title":  "Invalid Attribute",
-      "source": { "pointer": "data/attributes/first-name" },
+      "source": { "pointer": "/data/attributes/first-name" },
       "detail": "First name must contain at least three characters."
     }
   ]
 }
 ```
 
-Notice that the attribute `first-name` is dasherized.  Dasherized attributes are preferred
```

Comment:
Removed this paragraph because it's not error specific, and I wanted to keep this section as concise as possible, since it's already so long.